src/script-handlers/slotHandler.ts

```diff
@@ -48,6 +55,57 @@ export default function eventHandler(documentation: Documentation, path: NodePat
         }
         this.traverse(pathMember)
       },
+      visitJSXElement(pathJSX: NodePath<bt.JSXElement>) {
+        const tagName = pathJSX.node.openingElement.name
+        if (bt.isJSXIdentifier(tagName) && tagName.name === 'slot') {
+          const doc = documentation.getSlotDescriptor(getName(pathJSX))
+          doc.description = getDescription(pathJSX)
+        }
+        this.traverse(pathJSX)
+      },
     })
   }
 }
+
+function getName(pathJSX: NodePath<bt.JSXElement>): string {
+  const oe = pathJSX.node.openingElement
+  const names = oe.attributes.filter(
+    (a: bt.JSXAttribute) => bt.isJSXAttribute(a) && a.name.name === 'name',
+  ) as bt.JSXAttribute[]
+
+  const nameNode = names.length ? names[0].value : null
+  return nameNode && bt.isStringLiteral(nameNode) ? nameNode.value : 'default'
+}
+
+function getDescription(pathJSX: NodePath<bt.JSXElement>): string {
+  let indexInParent = -1
+  const siblings = (pathJSX.parentPath.node as bt.JSXElement).children
+  if (!siblings) {
+    return ''
+  }
+  siblings.forEach((node: bt.JSXElement, i: number) => {
+    if (node === pathJSX.node) {
```

Comment:
I think you can reduce this to an `.indexOf`

Reply:
I do not know if typescript will be ok with it.